### PR TITLE
ATR-237: fixed the code fix name, refactored the attribute, added can…

### PR DESCRIPTION
**Pull request by SENya1990**
…cellation support

### CODE REVIEW

File: src/Acuminator/Acuminator.Analyzers/StaticAnalysis/MissingTypeListAttribute/MissingTypeListAttributeFix.cs [hunk 1]
```diff
@@ -24,48 +24,61 @@ public class MissingTypeListAttributeFix : CodeFixProvider
 
         public override async Task RegisterCodeFixesAsync(CodeFixContext context)
         {
-            var root = await context.Document.GetSyntaxRootAsync().ConfigureAwait(false);
-            var node = (PropertyDeclarationSyntax)root.FindNode(context.Span);
+			context.CancellationToken.ThrowIfCancellationRequested();
```
Comment (dropsonic) [inline src/Acuminator/Acuminator.Analyzers/StaticAnalysis/MissingTypeListAttribute/MissingTypeListAttributeFix.cs]: tabs/spaces mix

Comment (SENya1990) [reply src/Acuminator/Acuminator.Analyzers/StaticAnalysis/MissingTypeListAttribute/MissingTypeListAttributeFix.cs]: fixed

**Comment by dropsonic** [comment]
@SENya1990 same thing as in the previous pull request - GitHub shows that the spaces were replaced with tabs

**Comment by SENya1990** [comment]
@dropsonic isn't it our code style formatting setting to use Keep tabs option and Smart Indent?

**Comment by dropsonic** [comment]
@SENya1990 I think so but in this case, the old formatting has been changed (in places where there were no changes at all), so it makes it extremely difficult to compare

**Pull request merged**